cache/cache.go

```diff
@@ -136,6 +141,46 @@ func collectIncludePaths(homeDir, projectDir string, cacheLevel cache.Level) ([]
 	return includePths, nil
 }
 
+/*
+If the indicator is version controlled in git and has changes, we create a copy of it with its original content.
+*/
+func prepareUnmodifiedIndicator(indicator string) string {
+	indicatorDir := filepath.Dir(indicator)
+	indicatorFile := filepath.Base(indicator)
+
+	cmd := exec.Command("git", "ls-files", "--error-unmatch", indicatorFile)
+	cmd.Dir = indicatorDir
```

Comment:
is this command working if dir is a not git project root dir?

Reply:
Yeah it works in a sense it fails just as well as if the file is not tracked